publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/config/RabbitMqPropertiesConfig.java

```diff
@@ -119,7 +119,7 @@ public static String readJasyptKeyFile(String jasyptKeyFilePath) {
             }
 
         } catch (IOException e) {
-            System.out.println("Could not read the jasypt key from " + jasyptKeyFilePath);
+            log.error("Could not read the jasypt key from the jasypt key file path: " + e.getMessage(), e);
```

Comment:
Why printing both exception.getMessage and exception message? print any one

Reply:
done the change as you suggested